EE_API_automation/pytest/src/support/planner_jsons/edit_wi_assignee.json

```diff
@@ -2,7 +2,6 @@
   "data": {
     "id": "$wi_id",
     "attributes": {
-      "system.description.markup": "Markdown",
```

Comment:
Why was this removed?

Reply:
The Markdown field in the request is optional while updating Assignee. Its already tested as part of PATCH description test.